src/service/navigation.js

```diff
@@ -355,8 +346,10 @@ export class Navigation {
    * @private
    */
   handleNavClick_(e, target, tgtLoc) {
-    /** @const {!Location} */
-    const curLoc = this.parseUrlDeprecated_('');
+    const baseHref = getMode().test && !this.isEmbed_
```

Comment:
Why `getMode().test`?

Reply:
Because we cannot properly [polyfill](https://github.com/ampproject/amphtml/blob/master/testing/fake-dom.js#L162-L170) an anchor tag's href parsing. But, this only matters in test, where the anchor's base url will be `localhost:8000`. Anywhere else, this will be the proper url of the page displaying the doc.

Reply:
Let's add this in a comment.